test/issue-437-test.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+const { FakeTimers, assert } = require("./helpers/setup-tests");
+
+describe("issue #437", function () {
+    it("should save methods of children instance", function () {
```

Comment:
```suggestion
    it("should save methods of subclass instance", function () {
```

Reply:
Not sure why you talk about child instances? Children of what, if so. This seems like a normal subclass relationship.